R/Rules-methods.R

```diff
@@ -326,6 +326,192 @@ setMethod("nextBest",
           })
 
 
+##' --------------------------------------------------
+##' NCRM method with loss function & overdose control
+##' --------------------------------------------------
+##' @describeIn nextBest Find the next best dose based on the NCRM method and
+##' loss function.
+##' @importFrom ggplot2 ggplot geom_bar xlab ylab ylim aes geom_vline
+##' geom_hline geom_point
+##' @importFrom gridExtra arrangeGrob
+##' @importFrom stats sd
+##' @importFrom utils stack
```

Comment:
not needed, see my comment above, instead just use the functions with the namespace prefix

Reply:
Removed as suggested